### ISSUE TITLE
Change database.default.php

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#367]
Please change the default database host to localhost (instead of 127.0.0.1) to comply with the grants suggested in INSTALL.txt. This actually gives issues on CentOS. 
Also, the default port of 8889 should really be 3306.
```
diff --git a/app/Config/database.default.php b/app/Config/database.default.php
index 6d0b90e..ee14792 100644
--- a/app/Config/database.default.php
+++ b/app/Config/database.default.php
@@ -62,12 +62,12 @@ class DATABASE_CONFIG {
        public $default = array(
                'datasource' => 'Database/Mysql',
                'persistent' => false,
-               'host' => '127.0.0.1',
+               'host' => 'localhost',
                'login' => 'db login',
-               'port' => 8889,
+               'port' => 3306,
                'password' => 'db password',
                'database' => 'misp',
                'prefix' => '',
                //'encoding' => 'utf8',
        );
-}
\ No newline at end of file
+}
```

**Comment by RichieB2B** [comment]
Ok, I dug a little deeper into this and it has to do with SELinux. When using a host of 'localhost' mysql libraries on Unix will actually use a socket. When set to 127.0.0.1 TCP/IP is used. On CentOS httpd is not allowed to make outbound TCP/IP connection unless you set:
```
setsebool -P httpd_can_network_connect on
```
I still think the database.default.php should use localhost.

**Comment by RichieB2B** [comment]
Fixed in 98f65aae096a1866e0ca640af7a8663b5590020e
